packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1194,6 +1220,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
     private formMetadata(): IContainerRuntimeMetadata {
         return {
+            ...this.createContainerMetadata,
             summaryFormatVersion: 1,
             disableIsolatedChannels: this.disableIsolatedChannels || undefined,
             gcFeature: this.garbageCollector.gcSummaryFeatureVersion,
```

Comment:
(nit, for @agarwal-navin) feels like this should be gcVersion, not gcFeature

Reply:
Yeah, it should be. It just adds back-compat concerns so I am hesitant to change it.